mobile/go.mod

```diff
@@ -204,6 +207,12 @@ replace github.com/lightninglabs/lightning-node-connect => ../
 // We want to format raw bytes as hex instead of base64. The forked version
 // allows us to specify that as an option. This is required for the indirect
 // taproot-assets dependency to function properly.
-replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
+replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.34.2-hex-display
 
-go 1.22.3
+// Force faraday to use the master version as we'll either get an ambigious import if we try use faraday v0.2.13-alpha
+// as an indirect import here instead of indirectly importing faraday/frdrpc v1.0.0, or get an unkown revision for the
+// frdrpc if we don't include this replace. Once faraday v0.2.14-alpha is released, this should be resolved and this
+// replace can be removed in favour of importing that.
+replace github.com/lightninglabs/faraday => github.com/lightninglabs/faraday v0.2.13-alpha.0.20241118202659-a3aba5b7ea49
```

Comment:
Had to include this replace which isn't ideal, especially since it's referencing the unreleased `master` version, but couldn't figure any other way of getting the dependencies to work.